src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,578 @@
+[output]  # newrelic_alert_policy.DemoPolicy will be created
+[output]  + resource "newrelic_alert_policy" "DemoPolicy" {
+[output]    + account_id     = (known after apply)
+[output]    + id         = (known after apply)
+[output]    + incident_preference = "PER_POLICY"
+[output]    + name        = "My Demo Policy"
+[output]   }
+[output]
+[output] Plan: 1 to add, 0 to change, 0 to destroy.
+[output]
+[output] ------------------------------------------------------------------------
+[output]
+[output] Note: You didn't specify an "-out" parameter to save this plan, so Terraform
+[output] can't guarantee that exactly these actions will be performed if
+[output] "terraform apply" is subsequently run.
+```
+
+Run `terragrunt apply`. After Terraform finishes process, check your New Relic account for the new Demo Policy in your New Relic account.
+</Step>
+
+<!-- ADDING TO YOUR CONFIGURATION -->
+
+<Step>
+
+## Adding to your configuration
+
+Now that you have a basic New Relic configuration created, you add configurations built in the [Getting Started with Terraform](/terraform/get-started-terraform) and [Terraform modules](/terraform/terraform-modules) guides in this step.
+
+<Callout variant="tip">
+If you haven't done the previous developer guides, you can copy the configurations for this section from the <a href="https://github.com/jsbnr/nr-terragrunt-intro-example">Terragraunt Intro Github Repo</a>.
+</Callout>
+
+Update the __main.tf__ to include the addtional terraform configurations. Update the emaiil address in the alert channel to your prefered email address:
+```hcl
+resource "newrelic_alert_policy" "DemoPolicy" {
+	 name = "My Demo Policy"
+	}
+
+resource "newrelic_alert_channel" "DemoChannel" {
+	 name = "My Demo Channel"
+	 type = "email"
+
+	 config {
+	 recipients = "your@email_address.com"
+	 include_json_attachment = "1"
+	 }
+	}
+
+	resource "newrelic_alert_policy_channel" "ChannelSubs" {
+	 policy_id = newrelic_alert_policy.DemoPolicy.id
+	 channel_ids = [
+	 newrelic_alert_channel.DemoChannel.id
+	 ]
+	}
+
+	module "HostConditions" {
+	 source = "git::https://github.com/jsbnr/demo-terraform.git"
+	 policyId = newrelic_alert_policy.DemoPolicy.id
+	 cpu_critical = 88
+	 cpu_warning = 78
+	 diskPercent = 68
+	}
+```
+
+You have added a New Relic alert channel, subscribed the DemoPolicy to the alert channel, and added a module hosted on Github.
+
+To clone the module from Github, run `terragrun init` and then run `terragrunt apply`. After Terraform is done processing, You will see your Demo Policy now has two conditions in your New Relic account and one alert channel attached.
+
+</Step>
+
+<!-- ADDING ENVIRONMENT VARIABLE -->
+
+<Step>
+
+## Using your Environment as a Terragrunt Variable
+
+Using Terragrant, you can add the name of the environment you are running to the name of the data you are creating, making your resource more identifiable in New Relic.
+
+In the root __terragrunt.hcl__ file, create a new input for 'env_name'
```

Comment:
```suggestion
In the root __terragrunt.hcl__ file, create a new input for `env_name`
```

Try to use inline code blocks where possible when talking about code so that its easy to connect the two.